src/main/java/Event.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
```

Comment:
you could consider putting these task classes into one folder and packaging them! it will make your root directory neater!